src/main/webapp/app/shared/metis/postings-reactions-bar/answer-post-reactions-bar/answer-post-reactions-bar.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="me-0 row">
```

Comment:
This component looks almost the same as post-reactions-bar.component.html. Is there any reason to have the two of them? I think it should be possible to create a generic component that is used instead of the two. This way if changes to the components are needed they will be done in one place only.

Reply:
Yes, I know what you mean. There will be another PR where tutors/instructors use the emoji reaction bar on posts to pin and archive posts, this feature was initially integrated within this PR and I later extracted it because the scope was to big. With this feature, that is only available on posts, the component you are relating to will be differ for posts and answer-posts. That's why I would like to keep the templates separate. I already have a `postings-reactions-bar.directive.ts` which covers the common typescript functionality for both posting types. I hope that makes sense to you :)